src/WPF/ArcGISRuntime.WPF.Samples/Samples/Map/SearchPortalMaps/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Name": "Search a portal for maps",
+  "SampleName": "SearchPortalMaps",
+  "Description": "This sample demonstrates searching a portal for web maps and loading them in the map view. You can search ArcGIS Online public web maps using tag values or browse the web maps in your account.",
+  "Instructions": "",
```

Comment:
Yeah, good suggestion. I updated the metadata.json and ReadMe.md files to include instructions about the OAuth settings.